presto-orc/src/main/java/com/facebook/presto/orc/StripeReader.java

```diff
@@ -185,6 +188,26 @@ public Stripe readStripe(StripeInformation stripe, AggregatedMemoryContext syste
         // read the file regions
         Map<StreamId, OrcInputStream> streamsData = readDiskRanges(stripe.getOffset(), diskRanges, systemMemoryUsage);
 
+        long minAverageRowBytes = 0;
+        for (Entry<StreamId, Stream> entry : streams.entrySet()) {
+            if (entry.getKey().getStreamKind() == ROW_INDEX) {
```

Comment:
I was looking into the stripe footer and turns out the stats is in the file footer; my bad; stripe stats is collected by the orc reader instead of stripe reader. So I would just put 0 for `minAverageRowBytes` here I assume? And let the orc reader use the stripe stats if it gets a 0 from the row group's `minAverageRowBytes`. Feel like if stats is scattered in both row group and stripe level, the code would be kinda ugly. Will send out a PR soon; please let me know if this is proper.